docs/Training-on-Amazon-SageMaker-RL.md

```diff
@@ -0,0 +1,283 @@
+## Training on Amazon SageMaker RL
+
+This page contains instructions for setting up a SageMaker RL Training job for training ML-Agents environments. For the notebook containing the complete code, see [Unity 3D Game with Amazon SageMaker RL](https://github.com/aws/amazon-sagemaker-examples/tree/master/reinforcement_learning/rl_unity_ray).
+
+## Overview of architecture
+
+SageMaker is a fully managed service that enables fast model development. It provides many built-in features to assist you with training, tuning, debugging, and model deployment. SageMaker RL builds on top of SageMaker, adding pre-built RL libraries and making it easy to integrate with different simulation environments. You can use built-in deep learning frameworks such as TensorFlow and PyTorch with various built-in RL algorithms from the [RLlib](https://docs.ray.io/en/latest/rllib.html) library to train RL policies. Infrastructures for training and inference are fully managed by SageMaker, so you can focus on RL formulation. SageMaker RL also provides a set of [Jupyter notebooks](https://github.com/aws/amazon-sagemaker-examples/tree/master/reinforcement_learning), demonstrating varieties of domain RL applications in robotics, operations research, finance, and more.
```

Comment:
this section is a bit too much marketing - can you instead create a link to Sagemaker RL instead?  The architecture diagram below is helpful though.